test/test_data.js

```diff
@@ -0,0 +1,45 @@
+export const luaStatement = `
+function merge(sources)
```

Comment:
The fact that my reference transpiling-result contains these functions means that further development risks breaking this test when the output Lua changes. Preferrably I would want to localize the statement part of the output and compare only that. I'll be continuing to try different approaches.